# fix issue due to bug in OSX Firewire stack
# libdc1394 author comments here:
# http://permalink.gmane.org/gmane.comp.multimedia.libdc1394.devel/517

Index: dc1394/macosx/capture.c
===================================================================
--- dc1394/macosx/capture.c	(revision 594)
+++ dc1394/macosx/capture.c	(working copy)
@@ -150,7 +150,7 @@
 
     for (i = 0; i < buffer->num_dcls; i++) {
         int packet_size = capture->frames[buffer->i].packet_size;
-        if ((buffer->pkts[i].status & 0x1F) != 0x11) {
+        if ((buffer->pkts[i].status & 0x1F) != 0x11 && (buffer->pkts[i].status & 0x1F) != 0) {
             dc1394_log_warning ("packet %d had error status %x",
                     i, buffer->pkts[i].status);
             corrupt = 1;
